site/src/components/RichParameterInput/RichParameterInput.tsx

```diff
@@ -79,6 +79,7 @@ const RichParameterField: React.FC<RichParameterInputProps> = ({
   parameter,
   ...props
 }) => {
+  const [parameterValue, setParameterValue] = useState(parameter.default_value)
```

Comment:
You should not use a state here since the state is controlled by Formik. It can causes some mismatched values.